src/components/SecondaryPanes/CommandBar.js

```diff
@@ -53,7 +53,8 @@ function getKey(action) {
 }
 
 function getKeyForOS(os, action) {
-  return KEYS[os][action];
+  const actions = KEYS[os] || KEYS.Linux;
```

Comment:
This var name is throwing a lint warning: `no-shadow: 'actions' is already declared in the upper scope.`

We should use `OS` (caps) as the var name, IMO.